package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "origintrail_node",
-    "version": "6.0.9",
+    "version": "6.0.10",
```

Comment:
6.0.10 or 6.0.11? I guess it makes sense to start with 10 in case we need hotfix we jump to 11 then